src/sonic-yang-models/yang-models/sonic-nat.yang

```diff
@@ -0,0 +1,281 @@
+module sonic-nat {
+    namespace "http://github.com/Azure/sonic-nat";
+    prefix snat;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-acl {
+        prefix sacl;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC NAT";
+
+    revision 2019-10-30 {
+        description
```

Comment:
Update the revision to reflect the latest date.

Reply:
Done.